src/main/java/duke/Task.java

```diff
@@ -0,0 +1,53 @@
+package duke;
+
+import java.lang.StringBuilder;
+
+/**
+ * Task abstract class that Deadline, Event and Todo inherit from.
+ */
+public abstract class Task {
+    boolean isDone;
+    String input;
+
+    Task(String input) {
+        this.isDone = false;
+        this.input = input;
+    }
+
+    public void setDone() {
+        this.isDone = true;
+    }
+//
+//    public String getStatusIcon() {
+//        return (isDone ? "[done]" : "[undone]" ); //return tick or X symbols
+//    }
```

Comment:
Your code is really clean so far :-) You might want to remove this if it is no longer needed.